components/renderer/Dockerfile

```diff
@@ -3,19 +3,8 @@ FROM node:23.7.0-alpine3.21
 LABEL maintainer="Quality-time team <quality-time@ictu.nl>"
 LABEL description="Quality-time PDF render service"
 
-RUN apk add --no-cache \
-    msttcorefonts-installer=3.8.1-r1 \
-    fontconfig=2.15.0-r1 \
-    font-droid=20200215-r3 \
-    font-noto=24.7.1-r0 \
-    freetype=2.13.3-r0 \
-    ttf-dejavu=2.37-r5 \
-    ttf-freefont=20120503-r4 \
-    ttf-liberation=2.1.5-r2 \
-    chromium=132.0.6834.83-r0 && \
-    rm -rf /var/cache/apk/* /tmp/* && \
-    update-ms-fonts && \
-    fc-cache -f
+RUN apk add --no-cache chromium=132.0.6834.83-r0 && \
+    rm -rf /var/cache/apk/* /tmp/*
```

Comment:
Because we're using `--no-cache`, there shouldn't be a need of the `rm` right? Or does chromium do something sneaky

Reply:
I checked: the image size is the same with or without the `rm`, so I'll remove the `rm` command.